src/core/core_structs/results_model.jl

```diff
@@ -1,13 +1,51 @@
-struct OpertationModelResults
+struct OperationModelResults
     variables::Dict{Symbol, DataFrames.DataFrame}
-    total_cost::Dict{Symbol,Float64}
+    total_cost::Dict{Symbol, Any}
     optimizer_log::Dict{Symbol, Any}
+    times::DataFrames.DataFrame
+
 end
 
-function get_variable(res_model::OpertationModelResults, key::Symbol)
+function get_variable(res_model::OperationModelResults, key::Symbol)
     return get(res_model.variables, key, nothing)
 end
 
-function get_optimizer_log(res_model::OpertationModelResults)
-    return res_model.optimizer_log
+function get_optimizer_log(res_model::OperationModelResults)
+    return res_model.optimizer_log 
+end
+
+function get_times(res_model::OperationModelResults, key::Symbol)
+    return res_model.times
 end
+
+# passing in name of folder path and the name of the folder
+ 
+function operation_model_read(path::AbstractString, directory::AbstractString) 
```

Comment:
Let's make sure this function reflects that the files being read are from the results. Something like 
```load_model_results```